src/services/game_state.py

```
@@ -0,0 +1,76 @@
+            SELECT game_state FROM game_states
+            WHERE player_id = %s
+            """,
+            (player.name,),
+        )
+        result = self.cursor.fetchone()
+        if result:
+            game_state_json = result[0]
+            game_state = json.loads(game_state_json)
+            return {
+                "player": Player.deserialize(game_state["player"]),
+                "shop": Shop.deserialize(game_state["shop"]),
```

Ruff (0.8.2)

Undefined name `Shop`

(F821)